Sprint-1/interpret/percentage-change.js

```diff
@@ -12,11 +12,22 @@ console.log(`The percentage change is ${percentageChange}`);
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
-
+  //four functions
+    //carPrice = Number(carPrice.replaceAll(",", ""));
+    //priceAfterOneYear = Number(priceAfterOneYear.replaceAll("," ""));
+    //replaceAll(",", "")
+    //replaceAll("," "")
 // b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
-
+   //line 5,the error is because of missing comma in replaceAll function
 // c) Identify all the lines that are variable reassignment statements
-
+  // const priceDifference = carPrice - priceAfterOneYear;
+  // const percentageChange = (priceDifference / carPrice) * 100;
```

Comment:
These are not "variable reassignment statements"; they are variable declarations.